packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -181,7 +181,11 @@ const Dropdown = React.forwardRef(function Dropdown(
         <ListBox.Menu {...getMenuProps()}>
           {isOpen &&
             items.map((item, index) => {
-              const itemProps = getItemProps({ item, index });
+              const itemProps = getItemProps({
+                item,
+                index,
+                disabled: item?.disabled,
```

Comment:
```suggestion
                disabled: item.disabled,
```

I think `item` should always be defined / an object, or is that not the case?

Reply:
@joshblack `item` is but `disabled` is optional and may not always be available

Reply:
@tay1orjones nice, if `item` will be always be defined then what is the purpose of using `?.`? I had thought that the intent of optional chaining was that it would short-circuit if the value on the left hand was null-ish. In this case, since `item` is always an object, it seems like it'd be the same as `item.disabled` since they'd both return `undefined`.

Reply:
@joshblack Yeah! Sorry I overlooked this, I could've sworn I ran into a problem where `item` wasn't available. The optional chain fixed it, but I can't figure out how to repro it. Could be from my initial explorations and unrelated to how I ended up implementing this. 

I just pushed up a commit removing them 👍